dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -371,6 +371,73 @@ def test_profiled_kurtosis(self):
         df = pd.concat([df1, df2, df3])
         self.assertAlmostEqual(0.3311739, num_profiler.kurtosis)
 
+    def test_bias_correction_option(self):
+        data = np.linspace(-5, 5, 11).tolist()
+        df1 = pd.Series(data)
+
+        data = np.linspace(-3, 2, 11).tolist()
+        df2 = pd.Series(data)
+
+        data = np.full((10,), 1)
+        df3 = pd.Series(data)
+
+        # Disable bias correction
+        options = FloatOptions(); options.bias_correction.is_enabled = False
+        num_profiler = FloatColumn(df1.name, options=options)
+        num_profiler.update(df1.apply(str))
+        self.assertAlmostEqual(10, num_profiler.variance)
+        self.assertAlmostEqual(0, num_profiler.skewness)
+        self.assertAlmostEqual(89/50 - 3, num_profiler.kurtosis)
+
+        df2_ints = df2[df2 == df2.round()]
+        num_profiler.update(df2.apply(str))
+        df = pd.concat([df1, df2_ints])
+        self.assertAlmostEqual(6.3125, num_profiler.variance)
+        self.assertAlmostEqual(0.17733336, num_profiler.skewness)
+        self.assertAlmostEqual(-0.56798353, num_profiler.kurtosis)
+
+        df3_ints = df3[df3 == df3.round()]
+        num_profiler.update(df3.apply(str))
+        df = pd.concat([df1, df2_ints, df3_ints])
+        self.assertAlmostEqual(4.6755371, num_profiler.variance)
+        self.assertAlmostEqual(-0.29622465, num_profiler.skewness)
+        self.assertAlmostEqual(0.099825352, num_profiler.kurtosis)
+
+    def test_bias_correction_merge(self):
+        data = np.linspace(-5, 5, 11).tolist()
+        df1 = pd.Series(data)
+
+        data = np.linspace(-3, 2, 11).tolist()
+        df2 = pd.Series(data)
+
+        data = np.full((10,), 1)
+        df3 = pd.Series(data)
+
+        # Disable bias correction
+        options = FloatOptions();
+        options.bias_correction.is_enabled = False
```

Comment:
do we have the test for `options.bias_correction.is_enabled = True`?

Reply:
By default, options.bias_correction.is_enabled is True, so all tests without explicitly disabling the option is testing for the bias-corrected values.

Reply:
i think @AnhTruong is asking if we have a merge that ensures when you add two with True it stays True.

Reply:
This merge is tested by default in test_profile_merge and other similar cases, but should an explicit assert be added to make sure that bias stays True?

Reply:
correct, we should validate that if two trues are merged, it stays true

Reply:
@andrew-yin is this handled?

Reply:
@JGSweets just pushed those changes